crates/solidity/outputs/cargo/tests/src/bindings_output/runner.rs

```diff
@@ -74,26 +80,23 @@ pub fn run(group_name: &str, test_name: &str) -> Result<()> {
             "failure"
         };
 
```

Comment:
I suggest using `__SLANG_BINDINGS_RENDER_GRAPHS__` instead of `RENDER_GRAPHS`. The unique string will make it easy to search/reference in other parts of the codebase.